src/utils.ts

```diff
@@ -55,28 +55,52 @@ export function findPackageRoot(start: string | string[]): string {
  *
  * Additional types supported:
  * - Buffer
+ * - Dates
  *
  * Currently, these are only used for operation inputs.
  * TODO: Use in other contexts where we perform serialization and deserialization.
  */
 
-export function DBOSReplacer(_key: string, value: unknown) {
-  return value;
-}
-
+const JSON_DATE_VALUE_PREFIX = 'date:'
```

Comment:
I think the prefix could be more unique so we don't mistakenly deserialize other strings. For example, `DBOSJsonDate` or something like that.

Reply:
Sure! 
I was waiting for this. 
Just name it!
Maybe add some of $%^@ as well.